templates/rc.erb

```diff
@@ -0,0 +1,263 @@
+#! /bin/sh
+#
+# rc
+#
+# Starts/stops services on runlevel changes.
+#
+# Optimization: A start script is not run when the service was already
+# configured to run in the previous runlevel.  A stop script is not run
+# when the the service was already configured not to run in the previous
+# runlevel.
+#
+# Authors:
+# 	Miquel van Smoorenburg <miquels@cistron.nl>
+# 	Bruce Perens <Bruce@Pixar.com>
```

Comment:
I did not check the script itself, but I'm asking the license question first. Our repos are basically under Apache 2.0. This script was not developed by us, so if we want and decide to include it, we should ask the authors for their permission to include and distribute their work under Apache 2.0